src/queue_manager.h

```diff
@@ -20,11 +20,13 @@ struct Index_Request {
   std::vector<std::string> args;
   bool is_interactive;
   std::string contents;  // Preloaded contents. Useful for tests.
+  optional<lsRequestId> id;
```

Comment:
`lsRequestId` itself can be `std::monostate` to indicate TypeScript null. Maybe `optional` is unnecessary